packages/contracts-bridge/contracts/BridgeRouter.sol

```diff
@@ -155,6 +170,41 @@ contract BridgeRouter is Version0, Router {
         emit Send(_token, msg.sender, _destination, _recipient, _amount, false);
     }
 
+    /**
+     * @notice Send tokens to connext on a remote chain
+     * @param _token The token address
+     * @param _amount The token amount
+     * @param _destination The destination domain
+     * @param _externalId A hash of transfer metadata for Connext flow
+     */
+    function xsend(
+        address _token,
+        uint256 _amount,
+        uint32 _destination,
+        bytes32 _externalId
+    ) external {
+        // validate inputs
+        require(_externalId != bytes32(0), "!id");
```

Comment:
An idea could be to move away from input validation, which increases the net gas costs for everyone. I would expect such checks to be conducted on the front end, as smart contracts should not be used directly.